c++/src/kj/async-io.c++

```diff
@@ -2768,6 +2769,36 @@ String CapabilityStreamNetworkAddress::toString() {
   return kj::str("<CapabilityStreamNetworkAddress>");
 }
 
+Promise<size_t> FileInputStream::tryRead(void* buffer, size_t minBytes, size_t maxBytes) {
+  size_t result = file.read(offset, arrayPtr(reinterpret_cast<byte*>(buffer), maxBytes));
```

Comment:
Any intention here of paying attention to the `minBytes` contract here? If not, a comment here might be helpful. If yes, then a TODO comment here would be good.

Reply:
This implementation correctly handles `minBytes`. The requirement is that we only return fewer than `minBytes` on EOF, and a file read will only return fewer bytes than requested on EOF.

Reply:
(I'll add a comment.)

Reply:
5cd52440cc39cb061a525f382d02dc4b53c6dcad

Reply:
Awesome ok.